src/lib.rs

```diff
@@ -141,6 +141,8 @@ pub use util::schnorr::{self, SchnorrSig, SchnorrSigError};
 pub use util::ecdsa::PrivateKey;
 #[deprecated(since = "0.26.1", note = "Please use `ecdsa::PublicKey` instead")]
 pub use util::ecdsa::PublicKey;
+#[allow(deprecated)]
```

Comment:
I kinda sucks that I have to have this to avoid a warning when this is just for re-exporting.